src/ReactWrapper.js

```diff
@@ -294,7 +294,14 @@ export default class ReactWrapper {
    * @returns {ReactWrapper}
    */
   simulate(event, ...args) {
-    this.single(n => Simulate[event](findDOMNode(n), ...args));
+    this.single(n => {
+      const eventFn = Simulate[event];
+      if (eventFn !== undefined) {
+        eventFn(findDOMNode(n), ...args);
+      } else {
+        throw new TypeError(`ReactWrapper::simulate() event '${event}' does not exist`);
+      }
```

Comment:
nit: Only 1 of these clauses will ever be hit at a time making the `else` superfluous. What about just doing early termination when it is undefined. Something more like:

```js
const eventFn = ...;
if (!eventFn) {
  throw Error
}

eventFn(...);
```

Writing this way ends up being a little clearer because the only clause is for if the `eventFn` is undefined. Otherwise a reader can assume that block doesn't exist and read over it. Like I said, it's nit, but simplifies